version.go

```diff
@@ -0,0 +1,10 @@
+package dbresolver
+
+// Version is the current release version.
+const Version = "2.1"
+
+// Commit: is the HEAD commit of this [Version]
+const Commit = "4d12938cb692ad29618ab3b38b05fc4540948d6a"
+
+// ReleaseTime is the UTC Time of the release of this [Version]
+const ReleaseTime = "2023-06-04T03:34:16Z"
```

Comment:
Why do we need this?

Reply:
It becomes easier when debug issues related particular versions. I am working on ISSUE Template that would need the version of dbresolver. 


But i need to clear the values...

Reply:
We can a have it a separate PR if required. I kept it in this realizing that this is first PR that fixes an issue reported.